bundles/org.openhab.binding.panasonicbdp/src/main/resources/OH-INF/thing/bdplayer.xml

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="panasonicbdp"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Panasonic Blu-ray Player Thing -->
+	<thing-type id="bd_player">
```

Comment:
Please adapt to current [naming convention](https://www.openhab.org/docs/developer/guidelines.html#naming-convention):
```suggestion
	<thing-type id="bd-player">
```